### DISCUSSION TITLE
field to store birthday for contacts

### DISCUSSION THREAD

**Discussion by vlado2222** [Divested-Mobile/DivestOS-Build#127] [Ideas] [answered]
Hi.

I would like to ask you to add a birthday field into the contacts app. Then you can use [Birthday adapter](https://f-droid.org/en/packages/org.birthdayadapter/) from F-Droid to display birthdays in your calendar.

Thanks!

**Comment by SkewedZeppelin** [comment]
out of scope

**Comment by SkewedZeppelin** [comment, marked as answer]
Coming soon to 19.1:
https://review.lineageos.org/c/LineageOS/android_packages_apps_Contacts/+/332610
